src/js/components/AppListComponent.jsx

```diff
@@ -396,92 +381,130 @@ var AppListComponent = React.createClass({
 
     var totalColumnSpan = 8;
 
+    var inlineDialog = null;
+
+    if (pageIsLoading) {
+      inlineDialog = (
+        <CenteredInlineDialogComponent title="Loading Applications..."
+          message="Please wait while applications are being retrieved." />
+      );
+    }
+
+    if (pageHasNoRunningApps) {
+      let message = "Do more with Marathon by creating and organizing " +
+        "your applications.";
+      inlineDialog = (
+        <CenteredInlineDialogComponent additionalClasses="muted"
```

Comment:
As a developer you need to know these additional classed. Should we make constants of them? Also we could go with the "severity"-naming like at the dialogs.